static/js/components/ProfileProgressControls.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import Button from 'react-mdl/lib/Button';
+
+import { saveProfileStep } from '../util/profile_edit';
+
+export default class ProfileProgressControls extends React.Component {
+  static propTypes = {
+    nextUrl: React.PropTypes.string,
+    prevUrl: React.PropTypes.string,
+    isLastTab: React.PropTypes.bool,
+    saveProfile: React.PropTypes.func.isRequired,
```

Comment:
is `saveProfile` actually being used anywhere? i can only see it being passed down to `ProfileProgressControls` as a prop, where it doesn't seem to be used